### PR TITLE
Add several tests for check-upgrade with advisories and minimal options

### PR DESCRIPTION
**Pull request by j-mracek**
Some of the tests are related to new option --minimal for check-upgrade

### CODE REVIEW

File: dnf-behave-tests/dnf/security-advisory.feature [hunk 1]
```diff
@@ -55,6 +98,15 @@ Scenario: upgrade-minimal sec-severity
         | Action        | Package                   |
         | upgrade       | advisory_B-0:1.0-2.x86_64 |
 
+Scenario: check-upgrade --minimal with sec-severity
+   When I execute dnf4 with args "upgrade-minimal --sec-severity Moderate"
```
Comment (kontura) [inline dnf-behave-tests/dnf/security-advisory.feature:102]: I believe the `I execute dnf4` step shouldn't be here, it is not being run, it is a wrong command and these tests are only for dnf5.

File: dnf-behave-tests/dnf/security-advisory.feature [hunk 2]
```diff
@@ -64,6 +116,15 @@ Scenario: upgrade-minimal with pkgs specified sec-severity
         | Action        | Package                   |
         | upgrade       | advisory_B-0:1.0-2.x86_64 |
 
+Scenario: check-upgrade --minimal with pkgs specified sec-severity
+   When I execute dnf4 with args "upgrade-minimal advisory_B --sec-severity Moderate"
```
Comment (kontura) [inline dnf-behave-tests/dnf/security-advisory.feature:120]: I believe the `I execute dnf4` step shouldn't be here, it is not being run, it is a wrong command and these tests are only for dnf5.

**Review by kontura — CHANGES_REQUESTED** [review]

File: dnf-behave-tests/dnf/security-advisory.feature (hunk 1 above)
Comment (j-mracek) [reply dnf-behave-tests/dnf/security-advisory.feature]: Thank you - I've replaced dnf4 and dnf5 by simple `dnf`

File: dnf-behave-tests/dnf/security-advisory.feature (hunk 2 above)
Comment (j-mracek) [reply dnf-behave-tests/dnf/security-advisory.feature]: fixed

**Review by kontura — APPROVED** [review]
Thank you

**Pull request merged**